docs/site/content/docs/latest/tips.md

```diff
@@ -0,0 +1,20 @@
+# Connect to Cluster Nodes with SSH
+
+You can use SSH to connect to individual nodes of management clusters or workload clusters. The SSH key pair that you created when you deployed the management cluster must be available on the machine on which you run the SSH command. Consequently, you must run `ssh` commands on the machine on which you run `tanzu` commands.
+
+The SSH keys that you register with the management cluster, and consequently that are used by any workload clusters that you deploy from the management cluster, are associated with the following user accounts:
+
+- vSphere management cluster and workload cluster nodes running on both Photon OS and Ubuntu: `capv`
+- Amazon EC2 bastion nodes: `ubuntu`
+- Amazon EC2 management cluster and workload cluster nodes running on Ubuntu: `ubuntu`
+- Amazon EC2 management cluster and workload cluster nodes running on Amazon Linux: `ec2-user`
+- Azure management cluster and workload cluster nodes (always Ubuntu): `capi`
+
+To connect to a node by using SSH, run one of the following commands from the machine that you use as the bootstrap machine:
+
+- vSphere nodes: `ssh capv@<em>node_address`
```

Comment:
What does the `<em>` mean here? If this is an HTML emphasis tag, that will not be formatted correctly in inside the code formatting markdown.

Should these also includer `-i [path_to_key_file]`?

Reply:
I think we need an update here... based on feedback

Reply:
removed the html
I'm not sure about adding -i [path_to_key_file], I think the key just needs to be on the machine